pkg/resource/plugin/host.go

```diff
@@ -387,6 +398,22 @@ func (host *defaultHost) GetRequiredPlugins(info ProgInfo, kinds Flags) ([]works
 	return plugins, nil
 }
 
+func (host *defaultHost) SignalCancellation() error {
+	var buf bytes.Buffer
+	for _, plug := range host.resourcePlugins {
+		if err := plug.Plugin.SignalCancellation(); err != nil {
+			buf.WriteString(fmt.Sprintf("%s\n", err.Error()))
```

Comment:
We usually use the go-multierror package for this sort of thing.

Reply:
I'd prefer if this function faithfully aggregated the errors and returned them. Then let the caller -- which for this particular case wants to ignore errors -- decide what to do.

Reply:
Not sure I fully understand the second message, but I've wrapped this in `multierror`, which is what I think you want...